unfoldingWord/translationCore
# Added translationWords, Fixed Many USFM Workflow Issues

#### This pull request addresses:
This PR address issues with changing a usfm book / target language, and imported the same type of project, as well as merge conflicts.


#### How to test this pull request:
Try importing two projects with the same book / target language combination, you should not be able to.
Try importing a project with no book, and then enter a book that is already matching a projects book / target language combination.
Try the above with and without merge conflicts.


---
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/unfoldingword-dev/translationcore/2971)
